app/(login)/LoginScreen.tsx

```diff
@@ -2,12 +2,14 @@ import styles from './styles'
 import { Text, View, } from '../../components/Themed';
 import { TextInput } from 'react-native-gesture-handler';
 import { ImageBackground, Pressable } from 'react-native';
-import { useState } from 'react';
 import { Link, router } from 'expo-router';
 import { MaterialCommunityIcons } from '@expo/vector-icons';
 import { LogoAndName } from '../../components/LogoAndName';
+import { useState } from 'react';
+import { useUser } from '../../hooks/useUser';
 
 export default function loginScreen () {
+  const user = useUser(gun);
```

Comment:
Would try to look into useRef or something, so we don't recreate the user object on every render